Sources/JavaScriptKit/JSValueConvertible.swift

```diff
@@ -192,3 +198,15 @@ extension Array where Element: JSValueConvertible {
         Swift.Array<JSValueConvertible>.withRawJSValues(self)(body)
     }
 }
+
+extension JSValueConvertible where Self: Encodable {
```

Comment:
These kinds of internal implementation shouldn't output log without explicit user configuration and I think JSValueConvertiable shouldn't be failable.